teps/0025-hermekton.md

```diff
@@ -91,6 +91,7 @@ See [this rationale](https://github.com/kubernetes/community/blob/master/contrib
 
 | Object | Field | Description |
 | --- | --- | --- |
+| Task |  spec.steps[*].ExecutionMode | Whether or not Steps of this Task should happen hermetically. This can be overridden on the TaskRun |
```

Comment:
this says it can be overridden on the task run - i think that means we'd want to list a new field here for the taskrun as well? afaik we don't have any step specific overrides at the taskrun level so I'm tempted to suggest we don't include that - BUT in [TEP-0094](https://github.com/tektoncd/community/blob/main/teps/0094-configuring-resources-at-runtime.md#design-details) we propose adding `stepOverrides`, maybe execution mode could be included there? ( + @lbernick )

Reply:
This isn't a per-step override, AIUI the described override behavior is the ~enclosing resource: Step -> Task -> TaskRun -> Pipeline -> PipelineRun can provide a global override of the execution mode to everything contained within.

Pipelines seem to have a mechanism analogous to per-Task overrides as well, so perhaps adding something to `stepOverrides` makes sense as well, but this is the first I've seen of that.  Is it implemented?

Reply:
ahh kk I see what you mean - no I don't think `stepOverrides` is implemented yet.  sounds like this is something we can leave for now and potentially add to `stepOverrides` later if we need it (and once it exists)